public/js/lib/devtools-sham/client/shared/key-shortcuts.js

```diff
@@ -141,7 +141,7 @@ KeyShortcuts.parseElectronKey = function (window, str) {
     key = ElectronKeysMapping[key];
     shortcut.keyCode = window.KeyboardEvent[key];
     // Used only to stringify the shortcut
-    shortcut.keyCodeString = key;
+    shortcut.key = key;
```

Comment:
Can you explain these changes more? We need to stay in sync with m-c, and this file comes from there. It might be best for you to make these changes there first and then copy the new file over.